engo.go

```diff
@@ -5,14 +5,20 @@ import (
 	"log"
 
 	"engo.io/ecs"
+	"engo.io/engo/act"
 )
 
 var (
 	// Time is the active FPS counter
 	Time *Clock
 
 	// Input handles all input: mouse, keyboard and touch
-	Input *InputManager
+	Input *InputMgr
```

Comment:
Hmm, using consistent suffixes (like Mgr) is helpful for someone maintaining the code, using  Manager is annoying for a maintainer (More typing and longer, more complex lines to read). 

Using the full word is only helpful for people reading the code for the first time, learning the meaning of a consistent set of suffixes happens fast.